lib/authc/OAuthBasicExchangeAuthenticator.js

```diff
@@ -47,6 +47,7 @@ OAuthBasicExchangeAuthenticator.prototype.authenticate = function authenticate(c
 
         result.forApiKey = apiKey;
         result.application = self.application;
+        result.expiration = self.ttl;
```

Comment:
I'm not so sure that this should be renamed to `expiration`, see my other comment on this PR.